### DISCUSSION TITLE
Is there sql query implementation instead of using some type of ORM?

### DISCUSSION THREAD

**Discussion by jt6677** [realworld-apps/realworld#812] [Q&A]
I looked through most of the implementations and could not find any backend implementation using sql query, like " SELECT username, email, bio, image FROM users WHERE user_id = $1".

I would like to compare my query against theirs if possible. If any one is aware of such sql query implementation, please point me to that direction. Thank you

**Comment by KostyaTretyak** [comment]
Hi, @jt6677. I think [I have](https://github.com/ditsmod/realworld) exactly what you are looking for. For example, see [DbService](https://github.com/ditsmod/realworld/blob/main/packages/server/src/app/modules/routed/users/db.service.ts).

**Reply by jt6677** [reply]
> Hi, @jt6677. I think [I have](https://github.com/ditsmod/realworld) exactly what you are looking for. For example, see [DbService](https://github.com/ditsmod/realworld/blob/main/packages/server/src/app/modules/routed/users/db.service.ts).

Thanks for reply. I was looking for [sql create table](https://github.com/jt6677/conduit-fullstack/blob/main/backend/business/data/schema/schema.go). I did not like using ORM.
